functionApp.bicep

```diff
@@ -0,0 +1,194 @@
+param subscriptionId string
+param name string
+param location string
+param use32BitWorkerProcess bool
+param ftpsState string
+param storageAccountName string
+param linuxFxVersion string
+param sku string
+param skuCode string
+param workerSize string
+param workerSizeId string
+param numberOfWorkers string
+param repoUrl string
+param branch string
+param oidcUserIdentityName string
+param hostingPlanName string
+param serverFarmResourceGroup string
+param alwaysOn bool
+
+var contentShare = 'attendance-backend-dev8660'
+
+resource name_resource 'Microsoft.Web/sites@2022-03-01' = {
+  name: name
+  kind: 'functionapp,linux'
```

Comment:
Since we are not paying for the server, and linux and windows thus have the same price, I think windows is the better platform. Windows has better KUDU (advanced tooling) support, and has just better support on Azure (not a surprise of course, Microsoft owns both Windows and Azure).